packages/ts/form/test/Validation.test.ts

```diff
@@ -275,6 +275,38 @@ describe('form/Validation', () => {
         expect(crossFieldError?.message).to.equal('cannot be the same');
       });
     });
+
+    it('record level cross field validation when the property is a string', async () => {
+      const byPropertyName = (value: string) => (error: ValueError<any>) => {
+        const propertyName = typeof error.property === 'string' ? error.property : binder.for(error.property).name;
+        return propertyName === value;
+      };
+
+      const recordValidator = {
+        validate(value: Order) {
+          if (value.customer.fullName === value.customer.nickName) {
+            return { property: 'customer.nickName' };
```

Comment:
Seems that this test is only checking with the absolute property path: it's a top-level validator that receives the entire binder’s value.

In addition to that, we need a test case with a relative property string. Something like that:

```typescript
const customerValidator = {
  validate(value: Customer) {
    if (Array.from(value.fullName).reverse().join('') === value.nickName) {
      return { property: 'nickName' };
    }

    return true;
  },
  message: 'cannot be anagram of full name',
};
binder.for(binder.model.customer).addValidator(customerValidator);

/* ... */

binder.for(binder.model.customer.nickName).value = 'oof';
await binder.validate().then((errors) => {
  const crossFieldError = errors.find(byPropertyName('customer.nickName'));
  expect(crossFieldError).not.to.be.undefined;
  expect(crossFieldError?.message).to.equal('cannot be anagram of full name');
});
```